utils/builMqlUrqlClient.ts

```diff
@@ -10,14 +10,19 @@ function cleanUrl(url: string) {
 export default function buildMqlUrqlClient({
   clientVersion,
   mqlUrl,
-  token
+  token,
+  headers,
 }: {
   clientVersion: string,
   mqlUrl: string,
   token?: string,
+  headers?: Record<string, string>
 }) {
   return createClient({
     url: cleanUrl(mqlUrl),
+    fetchOptions: {
+      headers,
```

Comment:
This may also require changes on the mql server to allow the new headers.

Reply:
Yes. In the mql server side, we will add logic to handle new headers :)